spec/Overview.html

```diff
@@ -1865,6 +1863,133 @@ <h4>The digest method</h4>
+                            </p>
+                          </li>
+                          <li>
+                            <p>
+                              [= promise/React =] to |next|:
+                            </p>
+                            <dl class="switch">
+                              <dt>If |next| was rejected with |reason|:</dt>
+                              <dd>
+                                <ol>
+                                  <li>
+                                    <p>
+                                      Reject |bytesPromise| with |reason|.
+                                    </p>
+                                  </li>
+                                </ol>
+                              </dd>
+                              <dt>If |next| was fulfilled with |value|:</dt>
+                              <dd>
+                                <ol>
+                                  <li>
+                                    <p>
+                                      If |value| is [= end of iteration =],
+                                      resolve |bytesPromise| with |bytes|
+                                      and terminate these steps.
+                                    </p>
+                                  </li>
+                                  <li>
+                                    <p>
+                                      Append the result of [= get a copy of the buffer source |
+                                      getting a copy of the bytes held by =] |value|
+                                      to |bytes|.
+                                    </p>
+                                  </li>
+                                  <li>
+                                    <p>
+                                      Perform |getValueSteps|.
+                                    </p>
+                                  </li>
+                                </ol>
+                              </dd>
+                            </dl>
+                          </li>
+                        </ol>
+                      </li>
+                      <li>
+                        <p>
+                          [= Queue a microtask =] to perform |getValueSteps|.
+                        </p>
+                      </li>
+                    </ol>
+                    <div class=note>
+                      <p>
+                        The implementation may wish to compute the hash digest
+                        incrementally, instead of waiting until all data is
+                        available, in order to conserve memory.
+                      </p>
+                      <p>
+                        Additionally, if the |iterator| returned by
+                        [= open an async sequence | opening =] |data|
+                        is the iterator defined by <a data-cite="streams#readablestream">`ReadableStream`</a>,
```

Comment:
I think this should work?
```suggestion
                        is the iterator defined by {{ReadableStream}}</a>,
```
You'll first have to add `streams` to the `xref` config at the top:
```diff
- xref: ['html', 'dom', 'webidl', 'infra', 'encoding'],
+ xref: ['html', 'dom', 'webidl', 'infra', 'encoding', 'streams'],
```

Reply:
Yeah, I had that before, but decided I didn't really want to pull in an xref for a single reference in a side note :sweat_smile: